docs/release_notes.rst

```diff
@@ -37,33 +37,72 @@ repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
 
 What's New in 1.2.0
-~~~~~~~~~~~~~~~~~~~
+-------------------
 
-*RTI Connector* 1.2.0 is built on `RTI Connext DDS 6.1.1 <https://community.rti.com/documentation/rti-connext-dds-611>`__.
+*Connector* 1.2.0 is built on `RTI Connext DDS 6.1.1 <https://community.rti.com/documentation/rti-connext-dds-611>`__.
 
 New Platforms
 ^^^^^^^^^^^^^
 
 RTI has validated that *Connector* can be used on macOS 11 (Big Sur) systems.
 
+
+New API makes it easier to query what version of Connector is being used
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. CON-92 
+
+A new API has been added that provides the caller with the version of *Connector*
+and the version of the native libraries being used.
+
+
+What's Fixed in 1.2.0
+---------------------
+
+Deleting same Connector object twice may have resulted in segmentation fault
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A segmentation fault may have occurred when the same *Connector* object was
+deleted twice. This issue has been resolved.
+
+[RTI Issue ID CON-209]
+
+
+Support added for handling large 64-bit integers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Support has been improved for both getting and setting large (greater than 2^53)
+64-bit values. See :ref:`section-access-64-bit-integers` for more information.
+
+[RTI Issue ID CON-190]
+
+
 Previous Releases
-~~~~~~~~~~~~~~~~~
+-----------------
+
+Version 1.1.1
+^^^^^^^^^^^^^
+*Connector* 1.1.1 is built on *RTI Connext DDS* 6.1.0.3, which fixes several
+bugs in the Core Libraries. If you want more details on the bugs fixed in 6.1.0.3,
+contact support@rti.com. These bugs are also fixed in
```

Comment:
I added this last sentence. It certainly explains things for me. But will customers be confused? Let's get @alexcamposruiz to weigh in, too.

Reply:
I think it's fine, let's see what Alex says

Reply:
It looks good.